storage/client/file/src/main/java/com/azure/storage/file/models/DeleteSnapshotsOptionType.java

```diff
@@ -14,12 +14,7 @@
     /**
      * Enum value include.
      */
-    INCLUDE("include"),
-
-    /**
-     * Enum value only.
-     */
-    ONLY("only");
+    INCLUDE("include");
```

Comment:
An enum with only one built-in value type? Is it required?

Reply:
This is generated from Swagger.

Reply:
See https://github.com/Azure/azure-sdk-for-java/blob/master/storage/client/file/swagger/file.json#L3365